editions/multiwikiserver/tiddlywiki.info

```diff
@@ -11,6 +11,18 @@
 		"tiddlywiki/snowwhite"
 	],
 	"build": {
+		"--mws-list-users": [
+			"--mws-list-users"
+		],
+		"mws-add-user": [
```

Comment:
At the moment there is no parameterisation of build commands, so I can't see how this one could be used in practice. Is it intended to be part of the documentation?